pkg/kubectl/cmd/util/factory.go

```diff
@@ -123,16 +124,16 @@ type Factory struct {
 }
 
 const (
-	RunV1GeneratorName                          = "run/v1"
-	RunPodV1GeneratorName                       = "run-pod/v1"
-	ServiceV1GeneratorName                      = "service/v1"
-	ServiceV2GeneratorName                      = "service/v2"
-	HorizontalPodAutoscalerV1Beta1GeneratorName = "horizontalpodautoscaler/v1beta1"
-	DeploymentV1Beta1GeneratorName              = "deployment/v1beta1"
-	JobV1Beta1GeneratorName                     = "job/v1beta1"
-	NamespaceV1GeneratorName                    = "namespace/v1"
-	SecretV1GeneratorName                       = "secret/v1"
-	SecretForDockerRegistryV1GeneratorName      = "secret-for-docker-registry/v1"
+	RunV1GeneratorName                     = "run/v1"
+	RunPodV1GeneratorName                  = "run-pod/v1"
+	ServiceV1GeneratorName                 = "service/v1"
+	ServiceV2GeneratorName                 = "service/v2"
+	HorizontalPodAutoscalerV1GeneratorName = "horizontalpodautoscaler/v1"
```

Comment:
Created a new generator HorizontalPodAutoscalerV1, left the old one HorizontalPodAutoscalerV1Beta1. Which one should be a default?